firstvoices/backend/urls.py

```diff
@@ -15,6 +16,7 @@
 # site-level APIs
 sites_router = NestedSimpleRouter(ROUTER, r"sites", lookup="site")
 sites_router.register(r"dictionary", DictionaryViewSet, basename="dictionary")
+sites_router.register(r"category", CategoryViewSet, basename="category")
```

Comment:
[minor] Would it make sense for the URL to be `/api/1.0/sites/test-site/categories/` instead?

Reply:
Also I noticed that this hasn't been added to the sites serializer, does it need to be?

Reply:
+1 to both of these